.github/workflows/build_test_validate.yml

```diff
@@ -13,7 +13,7 @@ jobs:
           name: build_test
         strategy:
           matrix:
-            node-version: [ 16.x]
+            node-version: [14.x, 16.x]
```

Comment:
```suggestion
            node-version: [14.x, 16.x, 18.x]
```

Reply:
this would indicate otherwise, https://github.com/AzureAD/microsoft-authentication-library-for-js/pull/4727
(release with 1.9.0)

Reply:
Done